docs/framework/get-started/system-requirements.md

```diff
@@ -17,7 +17,7 @@ The tables in this article provide the hardware, operating system, and software
 - .NET Framework 4.5 and its point releases (4.5.1 and 4.5.2).
 - .NET Framework 4.6 and its point releases (4.6.1 and 4.6.2).
 - .NET Framework 4.7 and its point releases (4.7.1 and 4.7.2).
-- .NET Framework 4.8
+- .NET Framework 4.8 and its point release (4.8.1).
```

Comment:
Do we need to add anything about ARM processors in this article? See https://github.com/dotnet/docs/issues/27533.

Reply:
Arm64 is discussed in the blog.  I don't think we need to call out anything specific here since architecture isn't discussed.  .NET is supported on all architectures in support.